sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -0,0 +1,378 @@
+A `TextAnalyticsClient` is the primary interface for developers using the Text Analytics client library.  It provides both synchronous and asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction. 
+
+### Text Input
+A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models in the Text Analytics service.  Operations on `TextAnalyticsClient` may take a single text input or a collection of inputs to be analyzed as a batch.
+
+### Operation Result
+An operation result, such as `AnalyzeSentimentResult`, is the result of a Text Analytics operation, containing a prediction or predictions about a single text input.  An operation's result type also may optionally include information about the input document and how it was processed.
+
+### Operation Result Collection
+ An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index the input and result collections.  An operation result collection may optionally include information about the input batch and how it was processed.
+
+ ### Operation Overloads
+ For each supported operation, `TextAnalyticsClient` provides method overloads to take a single text input, a batch of text inputs as strings, or a batch of `TextDocumentInput` objects.  The overload taking the `TextDocumentInput` batch allows callers to give each document a unique ID, or indicate that the documents in the batch are written in different languages.
+
+ ## Examples
+ The following section provides several code snippets using the `client` [created above](#create-textanalyticsclient), and covers the main functions of Text Analytics.
+
+### Sync examples
+* [Detect Language](#detect-language)
+* [Analyze Sentiment](#analyze-sentiment)
+* [Extract Key Phrases](#extract-key-phrases)
+* [Recognize Entities](#recognize-entities)
+* [Recognize PII Entities](#recognize-pii-entities)
+* [Recognize Linked Entities](#recognize-linked-entities)
+
+### Async examples
+* [Detect Language Asynchronously](#detect-language-asynchronously)
+* [Recognize Entities Asyncronously](#recognize-entities-asynchronously)
+
+### Detect Language
+Run a Text Analytics predictive model to determine the language that the passed-in input text or batch of input text documents are written in.
+
+```C# Snippet:DetectLanguage
+string input = "Este documento está en español.";
+
+// Detect the language the input text is written in
+DetectLanguageResult result = client.DetectLanguage(input);
+DetectedLanguage language = result.PrimaryLanguage;
+
+Console.WriteLine($"Detected language {language.Name} with confidence {language.Score:0.00}.");
+```
+
+Please refer to the service documentation for a conceptual discussion of [language detection][langugage_detection].
```

Comment:
Nice! I was trying to figure out where to insert some of these concept docs - this seems like a good place.